### PR TITLE
Documentation to update generated API reference documentation 

### PR DESCRIPTION
**Pull request by bobdonat**
This pull request documents the steps for updating the generated API reference documentation.

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -39,3 +39,21 @@ Access the local Hugo server to see the rendered docs.  The rendered docs will r
 ```
 open http://localhost:1313
 ```
+
+## Update Generated API Reference Documentation
+
+To update the generated API reference documentation follow these steps:
+
+1. Checkout a docs repo feature branch.
```
Comment (rosemarymarano) [inline DEVELOPMENT.md:47]: Checkout  -> Check out

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -39,3 +39,21 @@ Access the local Hugo server to see the rendered docs.  The rendered docs will r
...
+```
+git checkout -b <branch-name>
+```
+
+2. Run a `Makefile` target to generate the API reference documentation. 
+Specify the verrazzano repo branch to generate the documentation from.
```
Comment (rosemarymarano) [inline DEVELOPMENT.md:53]: Specify the verrazzano repo branch from which to generate the documentation.

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -39,3 +39,21 @@ Access the local Hugo server to see the rendered docs.  The rendered docs will r
...
+```
+make generate-api BRANCH=<branch-name>
+```
+
+3. If new versions of the API reference documentation are generated,then commit those changes, push the changes,
```
Comment (rosemarymarano) [inline DEVELOPMENT.md:58]: generated,then commit those changes, push the changes -> generated, (space) then commit and push the changes,

**Review by rosemarymarano — APPROVED** [review]
Few literary edits

File: DEVELOPMENT.md (hunk 1 above)
Comment (bobdonat) [reply DEVELOPMENT.md:47]: fixed

File: DEVELOPMENT.md (hunk 2 above)
Comment (bobdonat) [reply DEVELOPMENT.md:53]: fixed

File: DEVELOPMENT.md (hunk 3 above)
Comment (bobdonat) [reply DEVELOPMENT.md:58]: fixed

**Pull request merged**